raiden/tests/test_service.py

```diff
@@ -29,6 +29,32 @@ def test_ping(raiden_network):
     assert decoded.echo == sha3(ping.encode() + app1.raiden.address)
 
 
+@pytest.mark.timeout(3)
```

Comment:
What is this timeout here for? According to [pytest-timeout](https://bitbucket.org/pytest-dev/pytest-timeout/) documentation this would simply mark the entire test as timed out after 3 seconds, right?